src/renderer/store/modules/settings.js

```diff
@@ -286,6 +286,7 @@ const state = {
   settingsPassword: '',
   allowDashAv1Formats: false,
   commentAutoLoadEnabled: false,
+  deArrowTitles: false,
```

Comment:
Not highlighting other places to be updated
```suggestion
  useDeArrowTitles: false,
```